FreeRTOS/Test/CBMC/include/aws_freertos_ip_verification_access_ip_define.h

```diff
@@ -1,3 +1,3 @@
-eFrameProcessingResult_t publicProcessIPPacket( IPPacket_t * const pxIPPacket, NetworkBufferDescriptor_t * const pxNetworkBuffer ) {
-	prvProcessIPPacket(pxIPPacket, pxNetworkBuffer);
-}
```

Comment:
Is this PR modifying line endings? I don't see any change.

Reply:
Yes. It is. Because the patches depend on the line endings. Therefore I had to modify the line ending to make the patches work.

Reply:
I went ahead and made all line endings conform to the previous standards. This removed ~100 files from the changed list.